python/python/glide/async_commands/core.py

```diff
@@ -2699,6 +2699,70 @@ async def geosearch(
             await self._execute_command(RequestType.GeoSearch, args),
         )
 
+    async def geosearchstore(
+        self,
+        destination: str,
+        source: str,
+        search_from: Union[str, GeospatialData],
+        search_by: Union[GeoSearchByRadius, GeoSearchByBox],
+        count: Optional[GeoSearchCount] = None,
+        store_dist: bool = False,
+    ) -> int:
+        """
+        Stores the results of a geospatial search into a destination sorted set.
+
+        See https://valkey.io/commands/geosearchstore/ for more details.
+
+        Note:
+            When in cluster mode, both `source` and `destination` must map to the same hash slot.
+
+        Args:
+            destination (str): TThe key to store the search results.
+            source (str): The key of the sorted set representing geospatial data to search from.
+            search_from (Union[str, GeospatialData]): The location to search from. Can be specified either as a member
+                from the sorted set or as a geospatial data (see `GeospatialData`).
+            search_by (Union[GeoSearchByRadius, GeoSearchByBox]): The search criteria.
+                For circular area search, see `GeoSearchByRadius`.
+                For rectangular area search, see `GeoSearchByBox`.
+            count (Optional[GeoSearchCount]): Specifies the maximum number of results to store. See `GeoSearchCount`.
+                If not specified, stores all results.
+            store_dist (bool): Whether to store the distance from the center as the sorted set score. Defaults to False.
+                - The distance is from the center of the circle or box, as a floating-point number, in the same unit specified for that shape.
+                - If set to False, the geohash of the location will be stored as the sorted set score.
```

Comment:
Is geohash stored if `store_dist` set to True?

Reply:
If store_dist is false - the geohash is stored as score 
If it's true- the distance is stored as score

Reply:
Can you please update the doc? It was unclear for me.